### ISSUE TITLE
Pixel 5 returns wrong carrier frequency for L5 in GnssAntennaInfo

### ISSUE THREAD

**Issue by barbeau** [barbeau/gpstest#483]
**Describe the bug**

tl;dr - the Pixel 5 is returning the wrong carrier frequency for L5 in GnssAntennaInfo. I've confirmed via GPSTest and GnssLogger apps and also via the Android Studio debugger.

I've opened an issue on the AOSP tracker here with more details:
https://issuetracker.google.com/issues/179800215

This isn't an issue with GPSTest but I'll keep this open until it's fixed, as Pixel 5s are going to be reporting the wrong antenna CF values to the [GPSTest Device Database](https://docs.google.com/spreadsheets/d/1jXtRCoEnnFNWj6_oFlVWflsf-b0jkfZpyhN-BXsv7uo/edit#gid=0) until this is fixed.

**To Reproduce**
1. Clone https://github.com/barbeau/gpstest or download GnssLogger
1. Follow the steps at https://github.com/barbeau/gpstest/blob/master/LOGGING.md#data-output---csv to log the GnssAntennaInfo to CSV or JSON

For the code that registers the GnssAntennaInfo.Listener for callback in GPSTest, see https://github.com/barbeau/gpstest/blob/master/GPSTest/src/main/java/com/android/gpstest/GpsTestActivity.java#L1042.

**Expected behavior**
One GnssAntennaInfo element should have a carrier frequency of 1176.45 for it's support of L5 and E5a signals

**Observed behavior**
GnssAntennaInfo doesn't have a carrier frequency for L5/E5a, and instead has a carrier frequency of 1227.6, which is for L2, which the device doesn't support

**App, Device and Android version:** 

GPSTest v3.9.7 on Pixel 5 (Android 11, build RQA.210105.003)

**Comment by barbeau** [comment]
Looking at the GPSTest Device Database:
https://bit.ly/gpstest-device-database

...this same issue affects the Pixel 4a 5G and the Galaxy S21 Ultra (Snapdragon variant).

**Comment by barbeau** [comment]
Looks like this is fixed on the Pixel 5 RQ3A.210605.005 update that rolled out today - hasGnssAntennaInfo() now reports false instead of true, and the GnssAntennaInfo data isn't populated.

So this is better than before, in that bad data isn't reported, but we still don't have valid antenna data.

I'm going to close this as the Pixel 5 is now fixed, but https://github.com/barbeau/gpstest/issues/520 will remain open for the systemic issue across OEMs.
